saleor/graphql/shipping/mutations/shippings.py

```diff
@@ -305,7 +311,56 @@ def clean_input(cls, info, instance, data, input_cls=None):
                     )
                 }
             )
-        return cleaned_input
+
+    @classmethod
+    def clean_delivery_time(cls, instance, cleaned_input, errors):
+        """Validate delivery days.
+
+        - check if minimum_delivery_days is not higher than maximum_delivery_days
+        - check if minimum_delivery_days and maximum_delivery_days are positive values
+        """
+        min_delivery_days = cleaned_input.get("minimum_delivery_days")
+        max_delivery_days = cleaned_input.get("maximum_delivery_days")
+
+        if not min_delivery_days and not max_delivery_days:
+            return
+
+        error_occurred = False
+        if min_delivery_days and min_delivery_days < 0:
```

Comment:
What about personal pickup? It is not something that can have 0 as minimum delivery days?

Reply:
These fields are optional, I think for personal pickup you wouldn't fill any values here :)

Reply:
But you can define 0 delivery days, only under 0 values are forbidden.

Reply:
Oh, my bad. I was checking this before morning coffee and thought that this will raise an exception for min==0 :D 
Sorry!